rust/kcl-lib/src/execution/geometry.rs

```diff
@@ -1062,6 +1069,25 @@ impl AddAssign for Point3d {
     }
 }
 
+impl Sub for Point3d {
+    type Output = Point3d;
+
+    fn sub(self, rhs: Self) -> Self::Output {
+        Point3d {
```

Comment:
This should check units and convert `rhs`'s units to `self`'s units before subtracting.